auth0/src/main/java/com/auth0/android/authentication/storage/CredentialsManager.kt

```diff
@@ -113,6 +115,10 @@ public class CredentialsManager @VisibleForTesting(otherwise = VisibleForTesting
         if (scope != null) {
             request.addParameter("scope", scope)
         }
+        parameters.forEach { (key, value) ->
```

Comment:
all can be added at once. See https://github.com/auth0/Auth0.Android/blob/main/auth0/src/main/java/com/auth0/android/request/Request.kt#L35